### DISCUSSION TITLE
What is the release plan for SmallRye Open API 2.1.7 ?

### DISCUSSION THREAD

**Discussion by Emily-Jiang** [smallrye/smallrye-open-api#840] [General]
I am wondering when the next release is to happen as Open Liberty would like to consume this [fix](https://github.com/smallrye/smallrye-open-api/pull/829)

**Comment by phillip-kruger** [comment]
Hi @Emily-Jiang . 
We do not have a plan yet for a next release.
Our releases is usually very adhoc depending on the number of fixes 
Or if a runtime require it. Since you need it for openliberty, I can do a release tomorrow.
There is not much in the release ( that is ok) . Here the list https://github.com/smallrye/smallrye-open-api/milestone/46?closed=1

Would that work for you ?

**Comment by Emily-Jiang** [comment]
Thank you @phillip-kruger!  Sounds great!

**Comment by phillip-kruger** [comment]
@Emily-Jiang version 2.1.7 has been released. 
see https://github.com/smallrye/smallrye-open-api/releases/tag/2.1.7

**Reply by Emily-Jiang** [reply]
Thank you @phillip-kruger ! We will pull this into Open Liberty asap.

**Reply by codespearhead** [reply]
@Emily-Jiang This seems to be a solved discussion. [Can you close it](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion) so that it doesn't show up [in the discussion tab](https://github.com/smallrye/smallrye-open-api/discussions) as unanswered?
